examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -173,35 +50,26 @@ var _ = ginkgo.BeforeSuite(func() {
 	priv := ed25519.PrivateKey(privBytes)
 	factory := auth.NewED25519Factory(priv)
 
-	// Read the VM config in, so it can be passed in as JSON.
-	// ANR sometimes takes the param as a file path or JSON string
-	// and other times only a JSON string.
-	vmConfigData, err := os.ReadFile(vmConfigPath)
-	require.NoError(err)
-	vmConfig := string(vmConfigData)
-
-	e2e.CreateE2ENetwork(
-		context.Background(),
-		e2e.Config{
-			Name:                       consts.Name,
-			Mode:                       mode,
-			ExecPath:                   execPath,
-			NumValidators:              numValidators,
-			GRPCEp:                     gRPCEp,
-			NetworkRunnerLogLevel:      networkRunnerLogLevel,
-			AvalanchegoLogLevel:        avalanchegoLogLevel,
-			AvalanchegoLogDisplayLevel: avalanchegoLogDisplayLevel,
-			VMGenesisPath:              vmGenesisPath,
-			VMConfig:                   vmConfig,
-			SubnetConfigPath:           subnetConfigPath,
-			PluginDir:                  pluginDir,
+	gen := genesis.Default()
+	gen.WindowTargetUnits = fees.Dimensions{40000000, 450000, 450000, 450000, 450000}
+	gen.MaxBlockUnits = fees.Dimensions{1800000, 15000, 15000, 2500, 15000}
+	gen.MinBlockGap = 100
+	gen.CustomAllocation = []*genesis.CustomAllocation{
+		{
+			Address: prefundedAddrStr,
+			Balance: 10_000_000,
 		},
-	)
+	}
+	genesisBytes, err := json.Marshal(gen)
+	require.NoError(err)
 
-	e2e.SetWorkloadFactory(
-		&workloadFactory{factory: factory},
+	e2e.SetupTestNetwork(
```

Comment:
The structure I was thinking of to separate this was along the lines of:

./tests/e2e/e2e.go - contains the actual test coverage ie all of the `ginkgo.It` nodes and a function `SetWorkload(...)` to inject the dependency for generating tx sequences as needed
./tests/network/network.go - contains the current SynchronizedBeforeSuite and a function `Setup(...)` to set the parameters for constructing the network

To run this in VM specific e2e tests, it would then be roughly:

```golang
network.Setup(vmSpecificParams...)
e2e.SetWorkload(vmSpecificWorkloadParams...)
```

This ends up identical to what we have now, so probably better to leave this as is for now